generic_camera_node:
  ros__parameters:
    camera_calibration_file: file://config/camera.yaml
    file_name: "0"
    frame_id: camera
    image_height: 720
    image_width: 1280
    use_sim_time: false